.ci/deps.sh

```diff
@@ -82,28 +79,17 @@ fi
 # Change environment for flawfinder from python to python2
 sudo sed -i '1s/.*/#!\/usr\/bin\/env python2/' /usr/bin/flawfinder
 
-# Update hlint to latest version (not available in apt)
-if [[ -z "$(which hlint)" ]]; then
-  hlint_deb=$(ls -vr ~/.apt-cache/hlint_1.9.* 2>/dev/null | head -1)
-  if [[ -z "$hlint_deb" ]]; then
-    hlint_deb_filename=hlint_1.9.26-1_amd64.deb
-    # This is the same build as xenial hlint
-    hlint_deb_url="https://launchpad.net/ubuntu/+source/hlint/1.9.26-1/+build/8831318/+files/${hlint_deb_filename}"
-    hlint_deb=~/.apt-cache/$hlint_deb_filename
-    wget -O $hlint_deb $hlint_deb_url
-  fi
-  sudo dpkg -i $hlint_deb
-fi
-
-# cabal update to 1.22.9.0 and install ghc-mod 5.6.0
-if [[ -z "$(which ghc-mod)" ]]; then
-  cabal update && cabal install cabal-install-1.22.9.0
-  cabal install ghc-mod
-fi
-
 # NPM commands
-sudo rm -rf $(which alex)  # Delete ghc-alex as it clashes with npm deps
+if [[ -n "$(which alex)" ]]; then
+  # Delete ghc-alex as it clashes with npm deps
+  echo "Removing $(which alex)"
+  sudo rm -rf $(which alex)
+fi
 npm install
+if [[ -z "$(which alex)" ]]; then
```

Comment:
The reason is that after the first time, if `haskell-platform` isnt installed, the next time the npm alex is deleted.